src/main/java/com/dev/cinema/controllers/AuthenticationController.java

```diff
@@ -0,0 +1,24 @@
+package com.dev.cinema.controllers;
+
+import com.dev.cinema.model.dto.UserRequestDto;
+import com.dev.cinema.security.interfaces.AuthenticationService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class AuthenticationController {
+
+    private final AuthenticationService service;
+
+    @Autowired
+    public AuthenticationController(AuthenticationService service) {
+        this.service = service;
+    }
+
+    @PostMapping("/registration")
```

Comment:
here is checklist violation

Reply:
ok.    but i think i did the right thing?

Reply:
because we have register method